nntrainer/layers/time_dist.cpp

```diff
@@ -45,6 +49,11 @@ void TimeDistLayer::copy(std::shared_ptr<Layer> l) {
   this->dist_layer = from->dist_layer;
 }
 
+void TimeDistLayer::setDistLayer(std::shared_ptr<Layer> l) {
+  dist_layer = l;
+  Layer::setActivation(l->getActivationType());
```

Comment:
setFlatten as well?

Reply:
Yes. we support flatten like activation for some cases. I will update next pr.